### PR TITLE
feat: add support for meeting participants added/removed events

### PR DESCRIPTION
**Pull request by anhduc130**
## Description

Adds support to TeamsActivityHandler for meeting participants added/removed events.

## Specific Changes

- Added support for 2 new activity types - `application/vnd.microsoft.meetingParticipantsAdded` and `application/vnd.microsoft.meetingParticipantsRemoved`
- Added functions to register handlers for when teams meeting participants are added/removed.
- Added unit tests.

## Testing

To be added.

### CODE REVIEW

File: libraries/botbuilder/src/teamsActivityHandler.ts [hunk 1]
```diff
@@ -58,6 +60,52 @@ const TeamsMeetingEndT = z
...
     .nonstrict();
 
+const TeamsMeetingParticipantsAddedT = z
+    .object({
+        Id: z.string(),
+        JoinUrl: z.string(),
+        MeetingType: z.string(),
+        Title: z.string(),
+        ParticipantsAdded: z.array(
+            z.object({
```
Comment (joshgummersall) [inline libraries/botbuilder/src/teamsActivityHandler.ts:70]: This is pretty much identical to the participant structure below. Perhaps DRY this up by extracting to `ParticipantT`. Also may be worth throwing in `nonstrict()` on this object as well.

**Review by joshgummersall — COMMENTED** [review]
One small suggestion, otherwise looks good!

File: libraries/botbuilder/src/teamsActivityHandler.ts (hunk 1 above)
Comment (anhduc130) [reply libraries/botbuilder/src/teamsActivityHandler.ts]: Ah right, good catch. Thanks for the suggestion.

File: libraries/botbuilder/src/teamsActivityHandler.ts [hunk 2]
```diff
@@ -60,26 +60,28 @@ const TeamsMeetingEndT = z
     })
     .nonstrict();
 
+const ParticipantT = z
+    .object({
+        Id: z.string(),
+        Name: z.string(),
+        AadObjectId: z.string().optional(),
```
Comment (joshgummersall) [inline libraries/botbuilder/src/teamsActivityHandler.ts:67]: `optional()` only allows `undefined`, should these also be `nullable()`?

**Pull request closed without merging**